examples/stream_created_from_array_raw.ts

```diff
@@ -1,32 +0,0 @@
-import { createClient } from '@clickhouse/client'
```

Comment:
same as with `stream_created_from_array_json.ts`

Reply:
That's the only example of inserting serialized data. I'd keep it